src/Models/Hooks/useTimeSeriesData.ts

```diff
@@ -150,6 +150,8 @@ const getBulkADXQueryFromTimeSeriesTwins = (
         twins?.forEach((twin, idx) => {
             query += `${connection.kustoTableName} | where TimeStamp > ago(${agoTimeInMillis}ms)`;
             query += ` | where Id == '${twin.twinId}' and Key == '${twin.twinPropertyName}'`;
+            query += ` | extend Value = todouble(${ADXTableColumns.Value})`;
+            query += ` | where isnotnull(${ADXTableColumns.Value})`;
```

Comment:
I have couple of questions here:
1. why do we filter out the null values? Because it is important for chart rendering like to connect line between null values or how to show it in chart in general. So I am worrying that even if it is null it is still valid data for us? @darsney ?
2. should we change the order here if we use this null check like `| where isnotnull(${ADXTableColumns.Value})` is first then `| extend Value = todouble(${ADXTableColumns.Value})`. But if you try to catch values which are null after the todouble operation I think we should use the `Value` column instead?
3. Why do we extend table with a column named `Value`? I didn't see we use it in the project below? Was it intended?

Reply:
I’ll check in the am but I think todouble on a non numeric returns null so depending on if you want to filter out nulls - you have to do it after - not sure what todouble on a null does

Reply:
For #2 I think that makes sense to exclude the null values because if they send N/A and expect taht to count as "null" do we want to convert it to 0 or just no data? I feel like the chart would look very different for those two cases. If they send 0 and we parse it, then it would be 0 and we would get that value, so I feel like this _should_ only impact the string scenarios and this logic makes sense to me.

Reply:
cool, lets check what happens when the value is already `null`? Otherwise we will get bad request error due to query if `todouble(null)` throws error

Reply:
If the value is already null todouble does not throw an error and the result remains null.